ganga/GangaCore/GPIDev/Lib/Tasks/CoreTransform.py

```diff
@@ -111,6 +111,8 @@ def createUnits(self):
                     unit.name = "Unit %d" % len(self.units)
                     unit.inputdata = GangaDataset(
                         files=filelist[fid:fid + self.files_per_unit])
+                    unit.inputdata.treat_as_inputfiles = self.inputdata[
+                        0].treat_as_inputfiles
```

Comment:
strange line continuation here on the `[` was that intentional?

Reply:
Yes - its a copy and paste of what was there before. I guess its a PEP8 thing